tests/acceptance/test_project_information.py

```diff
@@ -90,27 +76,21 @@ def test_project_information_iso_format_playwright(
         date_value = "2023-05-24"
         time_value = "06:00:00"
 
-        # Locate inputs via aria-labels
-        name_sel = (
-            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_PROJECT_NAME)}"]'
-        )
-        date_sel = (
-            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_START_DATE)}"]'
-        )
-        time_sel = (
-            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_START_TIME)}"]'
-        )
+        # Locate inputs via test markers (see test_sections_and_flows usage)
+        name_sel = f'[test-id="{MARKER_PROJECT_NAME}"]'
+        date_sel = f'[test-id="{MARKER_START_DATE}"]'
+        time_sel = f'[test-id="{MARKER_START_TIME}"]'
 
-        _set_input_value(page, name_sel, project_name)
-        _set_input_value(page, date_sel, date_value)
-        _set_input_value(page, time_sel, time_value)
+        set_input_value(page, name_sel, project_name)
+        set_input_value(page, date_sel, date_value)
+        set_input_value(page, time_sel, time_value)
```

Comment:
Use `set_input_value_by_marker` from `playwright_helpers.py` as mentioned in other review comment.